Cargo.toml

```diff
@@ -1,10 +1,10 @@
 [workspace]
 members = [
     "vach",
-    "vach-assets-manager",
     "vach-cli",
     "vach-benchmarks"
 ]
+resolver = "2"
```

Comment:
Why do we need the new resolver?

Reply:
Why not

Reply:
But we're already using the 2021 edition, so we've been using the v2 resolver already.

Reply:
Oh ok